website/content/docs/api-gateway/api-gateway-usage.mdx

```diff
@@ -49,7 +49,7 @@ Refer to [Technical Specifications](/docs/api-gateway/tech-specs) for minimum so
    <CodeBlockConfig>
 
    ```shell-session
-   $ helm install consul hashicorp/consul --version 0.41.1 --values values.yaml
+   $ helm install consul hashicorp/consul --version 0.41.1 --values values.yaml --create-namespace --namespace consul
```

Comment:
There's another `kubectl apply` for the routes down below that I believe should specify `--namespace consul`

Reply:
Hmm... so do we actually want to suggest creating gateways and routes in the consul namespace? The controller makes sense since it's a resource that we manage with other Consul-managed components, but I would think that in practice Gateways/Routes should be created in namespaces that _aren't_ something like `consul` to have better application segmentation.

Reply:
That makes sense. I was on a Zoom call with Mike earlier and sort of confused about how we include the install for `routes.yaml` which doesn't actually exist 🤔

Reply:
Re `routes.yaml`, this came up previously in https://github.com/hashicorp/consul/pull/12416#pullrequestreview-891239916 (need to click to expand the collapsed notes) - we should probably consider a better way to restructure this at some point.